packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -15,11 +15,21 @@ import {
   valueToInt,
 } from './BaseWrapper'
 
+// TODO (amyslawson) is there a proper config to put this in?
+const DEFAULT_NUM_ATTESTATIONS_REQUIRED = 3
+const DEFAULT_ATTESTATION_THRESHOLD = 0.8
```

Comment:
Let's put these in the src/config.ts

Reply:
there is no contractkit/src/config.ts :/